src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/TagHelperSemanticRangeVisitor.cs

```diff
@@ -44,16 +44,26 @@ private void Visit(SyntaxList<RazorSyntaxNode> syntaxNodes)
         }
 
         #region HTML
+        public override void VisitMarkupTextLiteral(MarkupTextLiteralSyntax node)
```

Comment:
Step one if we're going to get rid of some TextMate tokens is we've got to do better about colorizing everything.